lib/build.ml

```diff
@@ -235,7 +236,7 @@ let list_revdeps t ~platform ~opam_version ~pkgopt ~base ~master ~after commit =
           | "" -> acc
           | revdep ->
               let revdep = OpamPackage.of_string revdep in
-              if OpamPackage.equal pkg revdep then
+              if List.exists (fun x -> OpamPackage.equal revdep x.PackageOpt.pkg) pkgs then
```

Comment:
@tmcgilchrist why was this included? Wasn't this a temporary commit? Why did you not keep the individual commits?

Reply:
Revert PR https://github.com/ocurrent/opam-repo-ci/pull/229